table/queries/merged_reduced_scans.sql

```diff
@@ -24,16 +24,20 @@ CREATE TEMP FUNCTION ClassifyError(error STRING,
                                    source STRING,
                                    template_match BOOL,
                                    blockpage_match BOOL,
-                                   blockpage_id STRING) AS (
+                                   blockpage_id STRING,
+                                   server_header STRING) AS (
   CASE
-    WHEN blockpage_match then CONCAT("content/blockpage:", blockpage_id)
+    WHEN blockpage_match THEN CONCAT("content/blockpage:", blockpage_id)
 
     # Content mismatch for hyperquack v2 which didn't write
     # content verification failures in the error field from 2021-04-26 to 2021-07-21
-    WHEN (NOT template_match AND (error is NULL OR error = "")) then "content/mismatch"
+    WHEN (NOT template_match AND (error IS NULL OR error = "")) THEN "content/mismatch"
```

Comment:
Won't this `content/mismatch` take precedence over the Akamai test?

Reply:
It wasn't in my data, but it might in the time period where we're missing hyperquack v2 error output. Moved it above.